aiohttp/worker.py

```diff
@@ -159,6 +161,11 @@ def _create_ssl_context(cfg):
             ctx.set_ciphers(cfg.ciphers)
         return ctx
 
+    def _fix_log_format(self, source_format, default_format):
+        if re.search(r'%\([^\)]+\)', source_format):
+            return default_format
```

Comment:
Ha, now I got you. The comment in issue was unclear to me. Going to fix this according to your input. Thanks!